src/test/java/seedu/jarvis/model/planner/DeadlineTest.java

```diff
@@ -0,0 +1,49 @@
+package seedu.jarvis.model.planner;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.jarvis.model.address.tag.Tag;
+
+
+
+class DeadlineTest {
```

Comment:
I also recommend that you include more tests for incorrect inputs to check for errors thrown.